### DISCUSSION TITLE
Response language

### DISCUSSION THREAD

**Discussion by gevrey** [n3d1117/chatgpt-telegram-bot#427] [Q&A]
From time to time, I send a prompt in English as a sound recording, but the bot replies to me in French (which is my mother tongue).

I have no idea why and do not want this. Where could that come from?

**Comment by AlexHTW** [comment]
Hey,

maybe your `ASSISTANT_PROMPT` is in French or you had a conversation with the bot in French before that was not reset.

**Comment by gevrey** [comment]
Hey 
ASSISTANT_PROMPT is not defined, and I often to reset.
There must be something else.
